public/scripts/models/githubRepos.js

```diff
@@ -3,15 +3,27 @@
 var application = application || {};
 
 (function(module) {
-  const githubRepos = {};
+  let githubRepos = {};
 
   githubRepos.all = [];
 
-  githubRepos.requestRepos = function(callback) {
-    $.get('/github/user/repos', function(response) {
+  githubRepos.requestRepos = function(ctx, next) { // ctx is a context object. next is the callback fxn
+    $.get('/github/user/repos?affiliation=owner,collaborator', function(response) {
       githubRepos.all = response;
     })
-    .then(callback);
+    ctx.githubRepos = githubRepos.all;
+    next();
```

Comment:
I am getting an error in the console when I try to click on your github link in your nav bar: Uncaught TypeError: fn is not a function
    at page.js:476
    at nextEnter (page.js:308)
    at githubRepos.requestRepos (githubRepos.js:15)
    at page.js:476
    at nextEnter (page.js:308)
    at page.js:477
    at nextEnter (page.js:308)
    at nextExit (page.js:296)
    at Function.page.dispatch (page.js:312)
    at Function.page.show (page.js:202)